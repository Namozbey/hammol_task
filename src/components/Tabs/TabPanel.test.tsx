import React from "react";
import { render } from "@testing-library/react";
import { TabPanel } from ".";

describe("Testing TabPanel", () => {
  it("Snapshots", () => {
    const { asFragment } = render(
      <TabPanel value="doc" current="doc">
        <p>Lorem ipsum dolor sit amet.</p>
      </TabPanel>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
