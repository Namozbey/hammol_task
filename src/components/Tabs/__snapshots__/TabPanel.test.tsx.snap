// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TabPanel Snapshots 1`] = `
<DocumentFragment>
  <p>
    Lorem ipsum dolor sit amet.
  </p>
</DocumentFragment>
`;
